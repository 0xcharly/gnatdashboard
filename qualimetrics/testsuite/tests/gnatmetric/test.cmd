# launch qualimetrics on a trivial project
qualimetrics -Pdefault

# verify that gnatmetric has saved metric in db
if [ `sqlite3 obj/qualimetrics.db "select count(id) from rules;"` -gt 0 ]
then
    echo "OK - rules saved"
else
    echo "NOK - no rules saved"
fi

# verify that gnatmetric tool has been saved in db
if [ `sqlite3 obj/qualimetrics.db "select count(id) from tools;"` -gt 0 ]
then
    echo "OK - tool saved"
else
    echo "NOK - tool not saved"
fi

