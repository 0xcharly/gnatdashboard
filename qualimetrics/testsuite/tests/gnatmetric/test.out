OK - rules saved
OK - tool saved
